packages/react/src/components/Tabs/Tabs.tsx

```diff
@@ -107,13 +110,28 @@ const Tabs = ({
 
   useDidUpdate(() => {
     focusedTabRef.current?.focus();
+    const event = new Event('change', { bubbles: true, cancelable: false });
+    focusedTabRef.current?.dispatchEvent(event);
```

Comment:
why use a custom event when we could just call their onChange handler manually here? Are you thinking it'd be nice for the reason of delegation or something?

To me, calling their function here is simpler/better

Reply:
Also, `change` is not a standard event for non form fields afaik. This could cause problems for apps using cauldron tabs who also have a useCapture `change` event

```js

document.body.addEventListener('change', e => {
  console.log('I expect e.target to be a checkbox or radio!')
}, true)
```

Reply:
One more thing - if we _do_ need to go the event route, would it not make sense to [fire a custom event instead](https://github.com/dequelabs/cauldron/issues/162)?

Reply:
This was only to have an `event` with the correct `target` property to match how `onChange` is typically called with form change events. I'm not opposed to just calling `onChange` directly if you think that's cleaner. I was originally thinking we might extend this later to allow `preventDefault`, but based on our slack conversation I don't think that's a route we would go down.

We could do `onChange: function({ activeTabIndex: number, target: HTMLLiElement })`, thoughts?

Reply:
> We could do onChange: function({ activeTabIndex: number, target: HTMLLiElement }), thoughts?

👍 I like that